.github/workflows/built-branch.yml

```
@@ -0,0 +1,65 @@
+name: Create a -built branch
+
+on:
+  workflow_call:
+    inputs:
+      php:
+        default: 8.0
+        required: false
+        type: number
+      node:
+        default: 16
+        required: false
+        type: number
+
+jobs:
+  built-branch:
+    name: Built Branch
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Clear previous vendor
+        shell: bash
+        run: rm -rf vendor
+      - name: Install dependencies
+        uses: php-actions/composer@v6
+        with:
+          php_version: ${{ inputs.php }}
+          version: 2
+          args: --prefer-dist --no-dev
+      - name: Check if the plugin has front-end assets
+        shell: bash
+        id: has-node-assets
+        run: |
+          [[ -f package.json ]] && echo "::set-output name=exists::true" || echo "::set-output name=exists::false"
+      - name: Setup Node
+        if: steps.has-node-assets.outputs.exists == 'true'
+        uses: actions/setup-node@v1
```

It seems there is a v3 available.

```suggestion
        uses: actions/setup-node@v3
```